hw/ip/kmac/doc/_index.md

```diff
@@ -53,7 +53,7 @@ Based on the given criteria, the MSG_FIFO depth was determined to store the inco
 
 The MSG_FIFO has a packer in front.
 It packs any partial writes into the size of inernal datapath (64bit) and stores in MSG_FIFO.
-It gives some freedome to the software not requiring to align the messages.
+It gives some freedom to the software not requiring to align the messages.
```

Comment:
As long as you're improving this, feel free to also tweak the grammar. Maybe "It frees the software from having to align the messages", or something like that.

Reply:
much better!